src/components/AboutUs/AboutUs.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import MemberCard from './MemberCard';
+import team from './team.json';
+
+const AboutUs = () => {
+  return (
+    <div className=" container mx-auto px-4 mt-28" >
+      <div style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        // margin: '25px 0'
+      }}>
+        <h1 className='text-4xl font-bold mt-0 mb-6'>
+          Meet the team
+        </h1>
+      </div>
+      <div className="grid gap-4 grid-cols-4">
```

Comment:
The grid here isn't responsive could be on a small screen divided into 2 or 1 column "grid-cols-1 sm:grid-cols-2 lg:grid-cols-4" here is a suggestion
also if you can increase the gap here a little bit

Reply:
I agree with Marwa

Reply:
most likely @dania-diar will handle this one.

Reply:
Ill merge this after responsiveness fix @karzan7899 @dania-diar

Reply:
> 

@Iseluin  i fixed the responsivness , could you please check it out now